bool finish_training
---
float64[] robot_state
float32[] lidar_measurements
float64 reward